frontend/src/Crossword.jsx

```diff
@@ -32,6 +48,69 @@ const Crossword = (props) => {
     setCanvas(canvas);
   }, []);
 
+  const cursorKey = useCallback(
+    (key) => {
+      let [r, c] = cursorRC;
+
+      if (key === "ArrowDown" || key === "Down") {
+        r += 1;
+      } else if (key === "ArrowUp" || key === "Up") {
+        r -= 1;
+      } else if (key === "ArrowLeft" || key === "Left") {
+        c -= 1;
+      } else if (key === "ArrowRight" || key === "Right") {
+        c += 1;
+      } else {
+        return false;
+      }
+
+      if (r >= 0 && r < squares.length && c >= 0 && c < squares[r].length)
+        setCursorRC([r, c]);
+      return true;
+    },
+    [cursorRC]
+  );
+
+  const escSwitchMode = (key) => {
+    if (key === "Escape" || key === "Esc") {
```

Comment:
Toggle text mode using the letter `T`?

First, this would prevent text mode and drawing mode to be activated simultaneously (currently possible).

Second, this would make switching to the text mode behave the same way as toggling between brush and eraser mode (consistency). `T` also seems to be the same shortcut as in Photoshop, making @rosasam happy.

Reply:
Already toggled with enter, see https://github.com/thesauri/korsord/pull/20/files/71300451dfdcb87e964fd66a35a850ff0d73a3c3#diff-1ec7bbb2d4f5a1c6526cbf95777c02d6R131.

Currently only one mode can be active at a time but drawing is always possible. Thought I/someone else would fix that at the same time as _Mouse in write mode should select square_ from the TODOs